R/assertions.R

```diff
@@ -1557,20 +1560,29 @@ assert_list_element <- function(list, element, condition, message_text, ...) {
   )
   if (any(invalids)) {
     invalids_idx <- which(invalids)
-    abort(
-      paste0(
+
+    # construct supplementary message listing elements that are not correct type
+    if (is.null(message)) {
+      info_msg <- glue_collapse(
+        glue(
+          "{{.code {arg_name}[[{invalids_idx}]]${element} =
+           {lapply(list[invalids_idx], `[[`, element)}}}"
+        ),
+        sep = ", ", last = ", and "
+      )
+      message <- c(
         message_text,
-        "\n",
-        paste0(
-          arg_name(substitute(list)),
-          "[[", invalids_idx, "]]$", element,
-          " = ",
-          lapply(list[invalids_idx], `[[`, element),
-          collapse = "\n"
-        )
+        i = paste("But,", info_msg)
```

Comment:
```suggestion
        i = paste(" But,", info_msg)
```
Am I crazy?
![image](https://github.com/pharmaverse/admiraldev/assets/10111024/d3db9144-1eae-42f9-94f3-f851dc6523a5)